services/gitea/tasks/create_user.yml

```diff
@@ -0,0 +1,64 @@
+- name: "Create Gitea user(s)"
+  uri:
+    url: '{{ gitea_api_url }}/admin/users'
+    method: POST
+    body_format: json
+    status_code: 201
+    force_basic_auth: yes
+    url_username: '{{ gitea_admin_user }}'
+    url_password: '{{ gitea_admin_passwd }}'
+    body:
+      username: '{{ user.name }}'
+      password: '{{ user.password }}'
+      email: '{{ user.email }}'
+      admin: '{{ "true" if user.admin else false }}'
+      must_change_password: false
+    return_content: true
+  until: _response.status == 201
+  register: _response
+  when: user.name not in _user_list
+
+- name: "Get Gitea existing repo(s)"
+  uri: 
+    url: '{{ gitea_api_url }}/user/repos'
+    method: GET
+    body_format: json
```

Comment:
you don't have a body

Reply:
I do and hands attatched to it. Otherwise would be unable to type.